recipes/gli/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conan import ConanFile
+from conan.tools.files import copy, get, load, save
+from conan.tools.layout import basic_layout
+from conan.tools.build import check_min_cppstd
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class GliConan(ConanFile):
+    name = "gli"
+    description = "OpenGL Image (GLI)"
+    topics = ("opengl", "image")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/g-truc/gli"
+    license = "LicenseRef-copying.txt"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("glm/cci.20230113", transitive_headers=True)
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
```

Comment:
In their readme, GLI lists the minimum compiler versions that it supports. Could you add support for that check here too? See https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L47 for an example on how to do it

Reply:
Done (hopefully correctly). Thanks.